### PR TITLE
D sminorcorrections

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Small corrections MS DS 5-10

### CODE REVIEW

File: DayrSuryan/DSEthiop7.xml [hunk 1]
```diff
@@ -286,12 +294,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <supportDesc>
                         <support>
                            <material key="paper"/>
-                           <p>The manuscript is made of an industrially produced Italian notebook
-                              (the title <foreign xml:lang="it">“Quaderno”</foreign>
-                              appears at the end, on <locus target="#iiv"/>).
-                              The notebook has been turned around for writing,
-                              the title <foreign xml:lang="it">“Quaderno”</foreign> now appears
-                              upside down. </p>
+                           <p>The manuscript is a re-utilized industrially produced Italian notebook.
```
Comment (thea-m) [inline DayrSuryan/DSEthiop7.xml:297]: reutilised?

**Review by thea-m — CHANGES_REQUESTED** [review]
orthographic question

File: DayrSuryan/DSEthiop7.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop7.xml:297]: ok. strange, New Oxford Style has re-embark, re-emerge, re-equip, re-establish. Perhaps we consider reutilise not just "utilise the second time", but "reconsider as smth new".

Comment (thea-m) [reply DayrSuryan/DSEthiop7.xml]: it might be right. but let's keep the s, re-utilised

**Review by thea-m — APPROVED** [review]

**Pull request merged**